src/redismodule.h

```diff
@@ -1022,6 +1040,27 @@ typedef struct RedisModuleTypeMethods {
     RedisModuleTypeAuxSaveFunc aux_save2;
 } RedisModuleTypeMethods;
 
+/* Key metadata class configuration structure.
+ * Must be aligned with KeyMetaConfAllVersions in module.c.
+ * See RM_CreateKeyMetaClass() documentation in module.c for detailed information. */
+typedef struct RedisModuleKeyMetaClassConfig {
+    uint64_t version;
+#define REDISMODULE_META_ALLOW_IGNORE 0
```

Comment:
What should be the value of flags if we don't want to allow ignoring the meta-data?